src/widget/form/chatform.cpp

```diff
@@ -1085,5 +1085,47 @@ void ChatForm::hideNetcam()
 
 void ChatForm::retranslateUi()
 {
+    QString volObjectName = volButton->objectName();
+    QString micObjectName = micButton->objectName();
     loadHistoryAction->setText(tr("Load chat history..."));
+
+    switch(callButton->objectName())
+    {
+    case QStringLiteral("green"):
+        callButton->setToolTip(tr("Start audio call"));
+        break;
+
+    case QStringLiteral("yellow"):
+        callButton->setToolTip(tr("Accept audio call"));
+        break;
+
+    case QStringLiteral("red"):
+        callButton->setToolTip(tr("End audio call"));
+    }
+
+    switch(videoButton->objectName())
+    {
+    case QStringLiteral("green"):
+        videoButton->setToolTip(tr("Start video call"));
+        break;
+
+    case QStringLiteral("yellow"):
+        videoButton->setToolTip(tr("Accept video call"));
+        break;
+
+    case QStringLiteral("red"):
+        videoButton->setToolTip(tr("End video call"));
```

Comment:
Just a general tip, when using switch-case statements:
Don't forget to `break;` in **every** case. And usually you want to define a `default: break;` as well.